src/pages/PrimeCalculator/Form/index.tsx

```diff
@@ -389,16 +370,19 @@ export const Form: React.FC = () => {
                     ? t('primeCalculator.stakedTokens.infos.label')
                     : undefined
                 }
+                // This value is incorrect: it should correspond to the amount of tokens taken in
+                // consideration when calculating the Prime rewards. It corresponds to the amount of
+                // tokens staked by the user, up to the maximum allowed
```

Comment:
This info conditionally appears if the user inputs more than `primeMaximumStakedXvsTokens`, which is the top cap of XVS staked taken into account, it is informed by the contract